src/main/java/dbService/dao/UsersDAO.java

```diff
@@ -0,0 +1,38 @@
+package dbService.dao;
+
+import dbService.executor.Executor;
+import dbService.usersData.UserProfile;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class UsersDAO {
+    private Executor executor;
+
+    public UsersDAO(Connection connection) {
+        executor = new Executor(connection);
+    }
+
+    public UserProfile getUserProfile(String login) throws SQLException {
+        return executor.execQuery("select * from users where user_name='"
```

Comment:
better change the name of the column in the db
```suggestion
        return executor.execQuery("select * from users where name = '"
```